### PR TITLE
main/snappy: upgrade to 1.1.7 & community/java-snappy: upgrade to 1.1.7.1

### PR DESCRIPTION
**Pull request by TBK**
According to the changelog a mistake was with 1.1.5 release so the SONAME/version changed it was fixed with a re-release as 1.1.6  https://abi-laboratory.pro/tracker/timeline/snappy/

I am still unsure if backwards compatibility is 100% or if the dependent packages need to be rebuild... :/

@jirutka I have tried to update your java-snappy patch to work 1.1.7.1 could to review it to ensure I have not made some silly mistake?

### CODE REVIEW

**Comment by TBK** [comment]
CI fails to use the version of snappy that is just build...

File: main/snappy/APKBUILD [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
-a086dbdfe2cb21419b33c2f8a2739224a8e843f2c6a51715100c723b9fbd9aed14bcfa791c20dae785b33ac1d1cf4812ed2c5c5d4058c5b8e56a0e004e9a8c60  fix-UnalignedCopy64.patch"
+doc() {
+	default_doc
+	pkgdesc="$pkgdesc documents"
+	mkdir -p "$subpkgdir"/usr/share/"$pkgname"
+	local i;
+	for i in AUTHORS CONTRIBUTING.md COPYING NEWS README.md \
+		format_description.txt framing_format.txt; do
+		cp "$builddir"/$i "$subpkgdir"/usr/share/"$pkgname"/
+	done
```
Comment (jirutka) [inline main/snappy/APKBUILD]: Documentation files should be installed into `/usr/share/doc/$pkgname`.

Please move the files to the desired location inside `package()` function and remove custom `doc()` function.

**Comment by TBK** [comment]
@jirutka Fixed.

**Comment by jirutka** [comment]
I've merged it in 5220530cc587c8cebc12e88b8013dddba10bca81 and 94048b3da99cb81a8bffa47bc212e58d465dd32a and triggered rebuild of dependent packages to be sure. Thanks!

**Pull request closed without merging**